src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,4 +10,6 @@ public class Messages {
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_INVALID_TASK_DISPLAYED_INDEX = "The task index provided is invalid";
+    public static final String MESSAGE_INVALID_SCORE = "The score provided is invalid";
+    public static final String MESSAGE_TASK_ALREADY_DONE = "The task is already completed";
```

Comment:
Should provide a way to amend the score of an already marked task. What if I typed in a wrong score?

Reply:
Unless the flow is to unmark and then mark again, which seems iffy

Reply:
There are two options to proceed with this:
- User can use the mark command to give score again; that means no exception if the task is already marked
- Create a update command to change the score if needed

Any views on this?

Reply:
i think we can just use the mark command again to give another score as it reduces the amount of commands bacause to make a new command just to give a new score seems like overkill imo